### PR TITLE
17 clean up motif tooltip

### PR DESCRIPTION
**Pull request by ciarareeve**
This pull request enhances the STR table and the interactive Plotly chart on the search page. Key updates include:

1. **Responsive Plotly Chart:**
   - The Plotly chart now adjusts to screen width changes, improving experience across devices.

2. **Table Replacement with Card-Based Layout:**
   - The original table is replaced with a card-based layout to fix formatting issues and improve click region control. The card layout is modular and flexible, making it easier to manage the appearance and behavior of each data item. This also prevents unintended interactions and allows for future functionality enhancements.

3. **Improved Tooltip/Popover Integration:**
   - Integrated Bootstrap popovers for additional information on hover.

### CODE REVIEW

**Review by gymreklab — CHANGES_REQUESTED** [review]
I tried this out and it works great! I made some minor cleanup requests for WebSTR.py then we can merge

File: WebSTR/WebSTR.py [hunk 1]
```diff
@@ -37,15 +37,23 @@ def search():
...
-    if region_data.shape[0] == 0: return render_template('view2_nolocus.html')
+    if region_data.shape[0] == 0: 
+        return render_template('view2_nolocus.html')
+    
     gene_trace, gene_shapes, numgenes = GetGeneShapes(region_query, region_genome, BASEPATH)
     plotly_plot_json, plotly_layout_json = GetGenePlotlyJSON(region_data, gene_trace, gene_shapes, numgenes)
+    
+    # Convert DataFrame to dictionary
+    data = region_data.to_dict(orient='records')
+    print("Region Data (dictionary format):", data)
```
[the remark this answers is not in the document]
Comment (ciarareeve) [reply WebSTR/WebSTR.py]: yes i think this was accidentally left over from part of a debug process.  just pushed changes.

File: WebSTR/WebSTR.py [hunk 2]
```diff
@@ -147,7 +155,8 @@ def main():
         server.config.update(
             TEMPLATES_AUTO_RELOAD=True
         )
-    server.run(debug=FLASK_DEBUG, host=args.host, port=args.port)
```
[the remark this answers is not in the document]
Comment (ciarareeve) [reply WebSTR/WebSTR.py:150]: Just set it back to FLASK_DEBUG and pushed.  Thank you!

**Review by gymreklab — DISMISSED** [review]
Thanks for making these changes!

**Review by gymreklab — APPROVED** [review]

**Pull request merged**